Templates (Project)/Hamburger/Views/Shell.xaml

```diff
@@ -29,8 +28,10 @@
 
         <Controls:HamburgerMenu.SecondaryButtons>
             <!--  settingspage button  -->
-            <Controls:HamburgerButtonInfo x:Name="SettingsButton" PageParameter="0"
-                                          PageType="views:SettingsPage">
+            <Controls:HamburgerButtonInfo x:Name="SettingsButton"
+                                          PageParameter="0"
+                                          PageType="views:SettingsPage"
+                                          ToolTipText="Settings">
```

Comment:
Since I am at the end, I am going to comment on one more thing. And that's doing this SAME thing with automation NAME. That's a very important one to carry through and I think it is reasonable that the tooltip would be DIFFERENT than the name in many scenarios. Not the simple ones, of course. Agree? Just another property. Name should be a string, but ToolTip should be object - maybe UIElement.